lib/tapioca/compilers/dsl/base.rb

```diff
@@ -1,6 +1,8 @@
 # typed: strict
 # frozen_string_literal: true
 
+require "parlour"
```

Comment:
Up to now, we relied on the generators that subclass from `Base` to require `parlour` but actually, this file has a direct dependency on `parlour` types as they are in the signatures. So we really should have had this require here all along.